lib/ykpiv.c

```diff
@@ -231,8 +260,7 @@ ykpiv_rc ykpiv_list_readers(ykpiv_state *state, char *readers, size_t *len) {
   }
 
   rc = SCardListReaders(state->context, NULL, readers, &num_readers);
-  if (rc != SCARD_S_SUCCESS)
-  {
+  if (rc != SCARD_S_SUCCESS) {
```

Comment:
Please drop this change, we can go back over unrelated changes when we've merged this.